### PR TITLE
Add xls and pdf merge support

### PR DESCRIPTION
**Pull request by feffes**
Requesting test results from @Swexbe

### CODE REVIEW

**Comment by feffes** [comment]
Fixes: #1, #2, #3

**Comment by ejohnsson6** [comment]
Tested for a few different cases with a .xls file straight from Fortnox. Output seems to be identical to Master branch.

[test_input.zip](https://github.com/feffes/verifyIT/files/3691881/test_input.zip)
[test_output.zip](https://github.com/feffes/verifyIT/files/3691879/test_output.zip)
[test_output_master.zip](https://github.com/feffes/verifyIT/files/3691898/test_output_master.zip)

File: src/main/kotlin/Main.kt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package main.kotlin
 import org.apache.pdfbox.pdmodel.PDDocument
+import org.apache.pdfbox.multipdf.PDFMergerUtility
 import org.apache.poi.ss.usermodel.DataFormatter
 import java.io.File
 import org.apache.poi.ss.usermodel.WorkbookFactory
```
Comment (feffes) [inline src/main/kotlin/Main.kt]: remove unused dependency

File: src/main/kotlin/Main.kt [hunk 2]
```diff
@@ -30,29 +31,26 @@ fun main(args : Array<String>) {
 
 
     val dataFormatter = DataFormatter()
```
Comment (feffes) [inline src/main/kotlin/Main.kt]: unused variable

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
...
-    * For example, you can upload it to Google Drive and download as xlsx.
+
+1. Download an xls from the verification list ~~and convert it to xlsx. (Would appreciate if someone would look at why it can't load the xls as it is)~~
+   ~~\* For example, you can upload it to Google Drive and download as xlsx.~~
 2. Compile with `mvn clean install`
-3. Run `java -jar target/target/verifyIT-1.0-SNAPSHOT-jar-with-dependencies.jar [input] [output]` (input is a valid xlsx file, output is a directory)
-4. Now your output directory should be filled with verifications in pdf format!
\ No newline at end of file
+3. Run `java -jar target/target/verifyIT-1.0-SNAPSHOT-jar-with-dependencies.jar [input] [output] [-A]` (input is a valid ~~xlsx~~ xls file, output is a directory, -A will merge the PDFs into one big file)
+4. Now your output directory should be filled with verifications in pdf format!
```
Comment (feffes) [inline README.md]: remove strikethrough tex

File: src/main/kotlin/Main.kt [hunk 4]
```diff
@@ -1,5 +1,6 @@
 package main.kotlin
 import org.apache.pdfbox.pdmodel.PDDocument
+import org.apache.pdfbox.multipdf.PDFMergerUtility
 import org.apache.poi.ss.usermodel.DataFormatter
```
Comment (feffes) [inline src/main/kotlin/Main.kt]: unused dependency

**Review by feffes — COMMENTED** [review]
Also remove both poi dependencies from pom.xml. They shouldn't be needed anymore. After that I think it should be merged. GJ @Swexbe !

File: src/main/kotlin/Main.kt (hunk 1 above)
Comment (ejohnsson6) [reply src/main/kotlin/Main.kt]: Solved by https://github.com/feffes/verifyIT/pull/4/commits/607c09982efa7af01a1b88add9e2d07aa567e032

File: src/main/kotlin/Main.kt (hunk 2 above)
Comment (ejohnsson6) [reply src/main/kotlin/Main.kt]: Solved by 607c099

File: README.md (hunk 3 above)
Comment (ejohnsson6) [reply README.md]: Solved by 7f16df3

**Pull request merged**